roundware/lib/api.py

```diff
@@ -698,8 +699,18 @@ def vote_asset(request, asset_id=None):
         raise RoundException("a vote_type is required for this operation")
     if not check_for_single_audiotrack(form.get('session_id')):
         raise RoundException(
-            "this operation is only valid for projects with 1 audiotrack")
+            "VOTE: this operation is only valid for projects with 1 audiotrack")
 
+    # determine user/voter from provided session_id
+    User = get_user_model()
+    device_id = models.Session.objects.filter(id=int(form.get('session_id'))) \
```

Comment:
Is there ever a case where device_id is empty? There is no condition to catch that.